windows-icon-handler/check-icon-handler/CheckIconHandler.cpp

```diff
@@ -0,0 +1,84 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
```

Comment:
Not the right header. This is the backend header, the whole `windows-icon-handler` folder should have the parsec core header.